### PR TITLE
Add ClaimChunk to list of plugins using PlaceholderAPI

### PR DESCRIPTION
**Pull request by cjburkey01**
### Type
- [ ] Internal change (Doesn't affect end-user).
- [ ] External change (Does affect end-user).
- [x] Wiki (Changes towards the [Wiki]).
- [ ] Other: __________ 

### Description
Adds my plugin, [ClaimChunk](https://github.com/cjburkey01/ClaimChunk), to the page for the list of plugins using PlaceholderAPI :)

[Wiki]: https://wiki.placeholderapi.com

### CODE REVIEW

File: docs/users/plugins-using-placeholderapi.md [hunk 1]
```diff
@@ -215,6 +215,9 @@ If your plugin isn't shown here and you want it to be added, [read the Wiki READ
 - **[CitizensBooks](https://www.spigotmc.org/resources/37465/)**
     - [x] Supports placeholders.
     - [ ] Provides own placeholders. [Link]
+- **[ClaimChunk](https://www.spigotmc.org/resources/claimchunk.44458/)**
+    - [x] Supports placeholders.
+    - [x] Provides own placeholders. [**[Link](https://github.com/cjburkey01/ClaimChunk/wiki/Placeholder-API-Integration)**]
```
Comment (Andre601) [inline docs/users/plugins-using-placeholderapi.md:220]: Please make an entry in the placeholder-list.md file and link to that instead of an external site.

**Review by Andre601 — CHANGES_REQUESTED** [review]

File: docs/users/plugins-using-placeholderapi.md (hunk 1 above)
Comment (cjburkey01) [reply docs/users/plugins-using-placeholderapi.md:220]: Ah my bad, thank you!

File: docs/users/plugins-using-placeholderapi.md [hunk 2]
```diff
@@ -215,6 +215,9 @@ If your plugin isn't shown here and you want it to be added, [read the Wiki READ
 - **[CitizensBooks](https://www.spigotmc.org/resources/37465/)**
     - [x] Supports placeholders.
     - [ ] Provides own placeholders. [Link]
+- **[ClaimChunk](https://www.spigotmc.org/resources/claimchunk.44458/)**
```
Comment (Andre601) [inline docs/users/plugins-using-placeholderapi.md:218]:
```suggestion
- **[ClaimChunk](https://www.spigotmc.org/resources/44458/)**
```

**Review by Andre601 — CHANGES_REQUESTED** [review]
URL sanitization needed, as per the [Wiki's README](https://github.com/PlaceholderAPI/PlaceholderAPI/tree/wiki?tab=readme-ov-file#wiki-structure).

**Review by Andre601 — APPROVED** [review]

**Pull request merged**